keps/sig-node/2403-pod-resources-allocatable-resources/README.md

```diff
@@ -172,6 +186,24 @@ the new proposed `GetAllocatableResources` API.
 
 Add additional tests to prove that unhealthy devices are skipped as part of GetAllocatable and empty NUMA topology is not returned.
 
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+##### Unit tests
+
+- `k8s.io/kubernetes/pkg/kubelet/api/podresources`: `20230530` - `68.6%`
+
+##### Integration tests
+
+N/A - node local feature covered by e2e test (`test/e2e_node`)
+
+##### e2e tests
+
+- `NodeFeature:PodResources`: https://storage.googleapis.com/k8s-triage/index.html?sig=node&test=NodeFeature%3APodResources
```

Comment:
there are some failures by this link. Are those expected?

Reply:
not really, even though they seem platform issue more than test/production code issue